src/types/modules/react-native-svg.d.ts

```diff
@@ -0,0 +1,24 @@
+import {StyleProp} from 'react-native';
+import {CommonPathProps as BaseCommonPathProps, SvgProps as BaseSvgProps} from 'react-native-svg';
+
+declare module 'react-native-svg' {
+    // eslint-disable-next-line @typescript-eslint/consistent-type-definitions
+    interface SvgStyle extends ViewStyle {
+        enableBackground?: string;
+    }
+
+    // eslint-disable-next-line @typescript-eslint/consistent-type-definitions
+    interface SvgProps extends BaseSvgProps {
+        xmlns?: string;
+        xmlnsXlink?: string;
+        xmlSpace?: string;
+        style?: StyleProp<SvgStyle>;
+        hovered?: string;
+        pressed?: string;
```

Comment:
Why all these props are needed?

Reply:
For some reason, when I augmented `react-native-svg` it started complaining about these values and them being undefined. If you remove the style prop, it starts showing errors related to these values, even without augmentation. I know it's weird. Do you have any ideas for possible causes?